lib/libzutil/zutil_import.c

```diff
@@ -1475,11 +1475,11 @@ slice_cache_compare(const void *arg1, const void *arg2)
 	uint64_t guid2 = ((rdsk_node_t *)arg2)->rn_vdev_guid;
 	int rv;
 
-	rv = AVL_CMP(guid1, guid2);
+	rv = AVL_ISIGN(strcmp(nm1, nm2));
```

Comment:
doesn't this change conflict with the comment block above?

Reply:
no, it still adds both.

Reply:
nit: should read "Sorted by full path then vdev guid to allow..."